Makefile

```diff
@@ -46,6 +46,8 @@ plugins:
 	go $(BUILD_TYPE) ./plugin/mysql
 	go $(BUILD_TYPE) ./plugin/rabbitmq-broker
 	go $(BUILD_TYPE) ./plugin/scality
+	go $(BUILD_TYPE) ./plugin/consul
+
```

Comment:
You're also going to need to add a command to the `release` target to get this into the BOSH release